solutions/parallel-routes-navbar/app/(demo)/demo/@breadcrumb/[category]/[slug]/page.tsx

```diff
@@ -0,0 +1,26 @@
+const DATA = {
+  'key-lime-pie': 'Key Lime Pie',
+  'pumpkin-pie': 'Pumpkin Pie',
+  'apple-pie': 'Apple Pie',
+}
+
+export default async function BreadcrumbPage({
+  params: { category, slug },
+}: {
+  params: { category: string; slug: string }
+}) {
+  const displayName = await new Promise<string>((resolve) =>
+    setTimeout(
+      () => resolve(DATA[slug as keyof typeof DATA] || 'Secret recipe'),
+      1000
+    )
+  )
```

Comment:
minor nit: I would move this as a `async function` above to simplify the function body.